packages/bitcore-wallet-client/test/api.test.js

```diff
@@ -4626,7 +4640,7 @@ describe('client API', function() {
             DATA = JSON.parse(TestData.payProJsonV2Body.btc);
 
             mockRequest(Buffer.from(TestData.payProJsonV2.btc.body, 'hex'), TestData.payProJsonV2.btc.headers);
-            helpers.createAndJoinWallet(clients, keys, 1, 1, x.opts, w => {
+            helpers.createAndJoinWallet(clients, keys, 2, 2, x.opts, w => {
```

Comment:
change tests to a 2-of-2 wallets so changes on segwit TX's vsize are more obvious